docs/proposals/202005_version_documentation.md

```diff
@@ -0,0 +1,164 @@
+---
+title: 'Building A Versioning Plugin For Thanos'
+type: proposal
+menu: proposals
+status: approved
+owner: thisisobate
+Date: May 2020
+---
+
+
+## Problem
+
+The Thanos website contains the docs area which is built by rendering the markdown files from the tip of master of the Thanos repository. This frequently causes confusion for users, as different Thanos versions have different features.
+
+In this proposal, we want to solve this. In particular, we want to:
+
+1. Build a Version Picker that serves as a tool which will aid easy access to other versions of the documentation. This picker will Include links to change version (the version must be in the URL).
+2. A solid and robust Documentation structure.
```

Comment:
```suggestion
2. A solid and robust documentation structure.
```

Reply:
what do we mean by `solid` here? `robust` we can understand in software as not prone to breaking easily, or withstanding different odd usages. Maybe:

```suggestion
2. A robust documentation structure.
```